### ISSUE TITLE
How to get fix shape of features from Roberta large model

### ISSUE THREAD

**Issue by DhruvMevada** [pytorch/fairseq#1725]
#### I'm getting a different feature shape. Is there any way to get the fixed shape of the extracted features from Roberta?

I anlysed that axis-0 and axis-2 gets fixed shape 1 and 1024 respectively. But axis-1 is changing based on the statement.

For "Hello world!" Roberta returns (1,5,1024)
For "Hello good morning Hello world!" Roberta returns (1,8,1024)

Is there any way to get the same shape of features from Roberta?

#### Code
```
In [1]: import torch 
   ...: roberta = torch.hub.load('pytorch/fairseq', 'roberta.large') 
   ...: roberta.eval()

In [2]: tokens = roberta.encode('Hello world!')                                 

In [3]: last_layer_features = roberta.extract_features(tokens)                  

In [4]: last_layer_features.shape                                               
**Out[4]: torch.Size([1, 5, 1024])**

In [5]: tokens = roberta.encode('hello good morning Hello world!')              

In [6]: last_layer_features.shape                                               
Out[6]: torch.Size([1, 5, 1024])

In [7]: last_layer_features = roberta.extract_features(tokens)                  

In [8]: last_layer_features.shape                                               
**Out[8]: torch.Size([1, 8, 1024])**
```

#### What's your environment?

 - fairseq Version (e.g., 1.0 or master):
 - PyTorch Version (e.g., 1.0) - 1.4.0
 - OS (e.g., Linux): Ubuntu 18
 - How you installed fairseq (`pip`, source): pip
 - Build command you used (if compiling from source): - 
 - Python version: python3.6 and python3.7
 - CUDA/cuDNN version: - 
 - GPU models and configuration: - 
 - Any other relevant information: -

**Comment by myleott** [comment]
We return the features of the entire last layer, which changes based on the input length (more input tokens -> more output features).

For classification tasks people typically use the so-called `[CLS]` features, which is just the first position of the last layer:
https://github.com/pytorch/fairseq/blob/4923f34790761f41170fd88cd06e4d00ab0c527c/fairseq/models/roberta/model.py#L230-L247

You can try something similar, but it's important to finetune the model in this setting, otherwise the `[CLS]` feature representation will be pretty bad.
